physics/Interstitials/UFS_SCM_NEPTUNE/GFS_MP_generic_post.F90

```diff
@@ -43,10 +43,10 @@ subroutine GFS_MP_generic_post_run(
       integer, intent(in) :: index_of_temperature,index_of_process_mp,use_lake_model(:)
       integer, intent(in) :: imfshalcnv,imfshalcnv_gf,imfdeepcnv,imfdeepcnv_gf,imfdeepcnv_samf
       integer, dimension (:), intent(in) :: htop
-      integer                                                :: dfi_radar_max_intervals
+      integer,                                 intent(in)    :: dfi_radar_max_intervals
```

Comment:
This variable isn't being used in the code as far as I can tell. Can we just take it out?

Reply:
Done.